ms-patches/reduce-allocation-merges.yml

```diff
@@ -11,4 +11,12 @@ title: Reduce Allocation Merges
   - Add a new Class to support the rematerialization of SR objects that are part of a merge;
   - Patch HotSpot to be able to serialize and deserialize debug information related to allocation merges;
   - Patch C2 to generate unique types for SR objects participating in some allocation merges.
+  - Fix: Incorrect rematerialization of scalar replaced objects in C2
+  - Fix: Segfault during deoptimization of C2-compiled code
+  - Fix: VM crash during heap dump after JDK-8287061
+  - Fix: AllocationMergesTests.java fails with "IRViolationException: There were one or multiple IR rule failures."
+  - Fix: Live Node limit exceeded limit after JDK-8316991
```

Comment:
NIT: word limit repeated in
"- Fix: Live Node limit exceeded limit after JDK-8316991"